code/modules/projectiles/guns/ballistic/shotgun.dm

```diff
@@ -48,6 +50,18 @@
 	sawn_desc = "Come with me if you want to live."
 	can_be_sawn_off  = TRUE
 
+/obj/item/gun/ballistic/shotgun/riot/sawoff(mob/user)
+	. = ..()
+	if(.)
+		weapon_weight = WEAPON_MEDIUM
+		inhand_icon_state = "gun"
+		worn_icon_state = "gun"
+		worn_icon = 'icons/mob/clothing/belt.dmi'
```

Comment:
already handled by the base proc and build_worn_icon

Reply:
Reverted. Updated the base sawoff proc to have the worn_icon_state set to "gun". Adding the inhands and to the list of missing sprites.